examples/calibrate_tsqrtP.py

```diff
@@ -0,0 +1,215 @@
+here the data is converted to analytic power, frequency filtered
+and the absolute is taken prior to integrating to return the beta where
+:math:`\beta = \frac{1}{\sqrt{2}} \int \sqrt{P(t)} dt` 
+"""
+import pyspecdata as psd
+import matplotlib.pyplot as plt
+import numpy as np
+from itertools import cycle
+
+colorcyc_list = plt.rcParams["axes.prop_cycle"].by_key()["color"]
+color_cycle = cycle(
+    colorcyc_list
+)  # this can be done more than once to spin up multiple lists
+
+V_atten_ratio = 102.35  # attenutation ratio
+skip_plots = 33  # diagnostic -- set this to None, and there will be no plots
+linear_threshold = 100e-6
+slicewidth = 3e6
+
+
+with psd.figlist_var() as fl:
+    for filename, nodename in [
+        # ("240805_calib_amp1_pulse_calib.h5", "pulse_calib_1"),  # high power
+        # ("240805_calib_amp0p1_a_pulse_calib.h5", "pulse_calib_3"),  # low power
+        # ("240805_calib_amp0p2_a_pulse_calib.h5", "pulse_calib_1"),  # low power
+        ("240805_calib_amp0p05_pulse_calib.h5", "pulse_calib_5"),  # low power
+    ]:
+        d = psd.find_file(
+            filename, expno=nodename, exp_type="ODNP_NMR_comp/test_equipment"
+        )
+        assert (
+            d.get_prop("postproc_type") is not None
+        ), "No postproc type was set upon acquisition"
+        amplitude = d.get_prop("acq_params")["amplitude"]
+        fl.basename = f"amplitude = {amplitude}"
+        if not d.get_units("t") == "s":
+            print(
+                "************ AG still needs to finish pyspecdata PR to save units!!! ************"
+            )
+            d.set_units("t", "s")  # why isn't this done already??
+        d *= V_atten_ratio
+        d /= np.sqrt(50)  # V/sqrt(R) = sqrt(P)
+
+        def switch_to_plot(d, j):
+            thislen = d["t_pulse"][j]
+            fl.next(f"pulse length = {thislen}")
+
+        def indiv_plots(d, thislabel, thiscolor):
+            if skip_plots is None:
+                return
+            for j in range(len(d["t_pulse"])):
+                if j % skip_plots == 0:
+                    switch_to_plot(d, j)
+                    fl.plot(
+                        d["t_pulse", j],
+                        alpha=0.2,
+                        color=thiscolor,
+                        label=thislabel,
+                    )
+
+        indiv_plots(d, "raw", "blue")
+        # {{{ data is already analytic, and downsampled to below 24 MHz
+        indiv_plots(abs(d), "analytic", "orange")
+        d.ft("t")
+        # {{{ apply frequency filter
+        d.ift("t")
+        SW = 1 / (d["t"][1] - d["t"][0])  # sample rate
+        carrier = (
+            d.get_prop("acq_params")["carrierFreq_MHz"] * 1e6
+        )  # signal frequency
+        n = np.floor(
+            (carrier + SW / 2) / SW
+        )  # how far is the carrier from the left side of the spectrum (which is at SW/2), in integral multiples of SW
+        nu_a = (
+            -SW / 2 + (carrier + SW / 2) - n * SW
```

Comment:
The SW/2 here cancel, and it makes equal sense without (you are just subtracting an integral SW, which is fine)